prow/config/jobs/istio.yaml

```diff
@@ -108,6 +108,19 @@ jobs:
       - name: TEST_SELECT
         value: "-postsubmit,-flaky,-multicluster"
 
+  - name: integ-operator-controller-tests
+    type: presubmit
+    command: [entrypoint, prow/integ-suite-kind.sh, test.integration.operator.kube.presubmit]
+    requirements: [kind]
+    modifiers: [optional]
+    env:
+      - name: HUB
+        value: "gcr.io/istio-testing"
+      - name: TAG
+        value: "1.6-dev"
```

Comment:
don't set this, otherwise we are not testing the PR and instead are testing the already merged code

Reply:
oh yes, but how can I specify hub and tag for this case? should I just leave it empty the test fw would infer?

Reply:
the test will set hub and tag to `istio-testing` (not a real hub) and build images locally. Then the jobs will use those local images

Reply:
got it